app/controllers/api/v2/hostgroups_controller_extensions.rb

```diff
@@ -0,0 +1,70 @@
+module Api
+  module V2
+    # Extends the hostgroups controller to support playing ansible roles
+    module HostgroupsControllerExtensions
+      include ForemanTasks::Triggers
+
+      # TODO: How to add this to API v2 controller?
```

Comment:
You need to do 2 things on both of these controllers:

* Add `extend ActiveSupport::Concern` to the module
* Wrap the methods in `included do end`. See an example of this [here](https://github.com/Katello/katello/blob/master/app/controllers/katello/concerns/api/v2/hostgroups_controller_extensions.rb) or [here](https://github.com/Katello/katello/blob/master/app/controllers/katello/concerns/api/v2/hosts_controller_extensions.rb)